theme/less/_header.less

```diff
@@ -0,0 +1,64 @@
+header
+{
+  box-shadow: 0px 8px 15px rgba(0, 0, 0, 0.06);
+}
+
+header label select
+{
+  color: @secondary;
+  font-size: 24px;
+  border: 0;
+  cursor: pointer;
+  margin: 25px;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background: url(/images/language-arrow.svg) transparent no-repeat 90% 50%;
+}
+
+header select option
```

Comment:
Meeting notes: We agreed not to style option

Reply:
Done.